src/components/avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import * as renderer from 'react-test-renderer';
```

Comment:
What do you think about removing `react-test-renderer` and this test? DOM snapshots can be useful in some situations, but I'm hoping to utilize `@testing-library/react` and image snapshots to avoid DOM snapshots.